src/xtgeo/common/calc.py

```diff
@@ -54,6 +53,57 @@ def ijk_to_ib(i, j, k, nx, ny, nz, ibbase=0, forder=True):
     return ib
 
 
+def xyori_from_ij(
+    iind: int,
+    jind: int,
+    xcor: float,
+    ycor: float,
+    xinc: float,
+    yinc: float,
+    ncol: int,
+    nrow: int,
+    yflip: int,
+    rotation: float,
+) -> Tuple[float, float]:
+    """Get xori and yori given X Y, geometrics and indices for regular maps/cubes.
+
+    Args:
+        iind: I index (zero based)
+        jind: J index (zero based)
+        xcor: X coordinate
+        ycor: Y coordinate
+        xinc: X increment (in non-rotated space)
+        yinc: Y increment (in non-rotated space)
+        ncol: Number of columns
+        nrow: Number of rows
+        yflip: YFLIP (handedness) indicator, 1 og -1
+        rotation: Rotation in degrees, anticlock from X axis
+
+    """
+
+    if iind >= ncol or iind < 0 or jind >= nrow or jind < 0:
+        raise ValueError("Indices out of range, exceeding ncol or nrow")
```

Comment:
Not a big deal, but would be good to inject which index is out of range

Reply:
OK